spec/lib/open_food_network/xero_invoices_report_spec.rb

```diff
@@ -98,15 +99,15 @@ module OpenFoodNetwork
 
       describe "when no initial invoice number is given" do
         it "returns the order number" do
-          subject.send(:invoice_number_for, order, 123).should == 'R731032860'
+          expect(subject.send(:invoice_number_for, order, 123)).to eq('R731032860')
         end
       end
 
       describe "when an initial invoice number is given" do
-        subject { XeroInvoicesReport.new user, {initial_invoice_number: '123'} }
+        subject { XeroInvoicesReport.new user, initial_invoice_number: '123' }
 
         it "increments the number by the index" do
-          subject.send(:invoice_number_for, order, 456).should == 579
+          expect(subject.send(:invoice_number_for, order, 456)).to eq(579)
         end
       end
     end
```

Comment:
So nothing in this spec actually changed right? You've just updated the syntax?

Reply:
Right. I wanted to run the spec on its own and had to modify it. And since I was at it, I just gave it the automated groom. I didn't have to change any logic though.